packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx

```diff
@@ -25,21 +22,11 @@ import { useSanitizePlaylistLibrary } from './useSanitizePlaylistLibrary'
 const { getPlaylistLibrary } = accountSelectors
 const { saveCollection } = collectionsSocialActions
 
-const messages = {
-  header: 'Playlists'
-}
-
 const acceptedKinds: DragDropKind[] = ['playlist']
 
-type PlaylistLibraryProps = {
-  scrollbarRef: MutableRefObject<HTMLElement | null>
-}
-
-export const PlaylistLibrary = (props: PlaylistLibraryProps) => {
-  const { scrollbarRef } = props
+export const PlaylistLibrary = () => {
```

Comment:
was the drag + drop functionality brought back somewhere? or what's the plan for that? future PR?

we currently allow you to drag a track onto the library and drag a playlist onto the playlist section. not sure whether we intend to keep that behavior or not with these nav changes

Reply:
yea the drag n drop logic is still in here, wasn't removed. this scrollbarRef was just used on a tooltip